index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+class Database {
+    private $conn;
+
+    public function __construct() {
+        $this->conn = new mysqli("localhost", "root", "password", "my_database");
```

Comment:
Use environment variables to store sensitive information:
$this->conn = new mysqli(getenv('HOST'), getenv('USER'), getenv('PASS'), getenv('DB_NAME'));